content/en/apps/reference/forms/app.md

```diff
@@ -199,6 +199,18 @@ To instruct the widget to process an array of strings or numbers, create a new `
 
 ## CHT XPath Functions
 
+### `add-date`
+
+_Available in +4.0.0._
+
+Adds the provided number of years/months/days/hours/minutes to a date value. 
+
+```
+add-date(date, years, months, days, hours, minutes)
+```
+
+This function is useful things like for calculating a person's birthdate date based on how many years/months old they are: `add-date(today(), 0-${age_years}, 0-${age_months})`. 
```

Comment:
I think this can be made clearer by stating that you can add negative numbers to get a date in the past. I'm thinking something like

```This function is useful for calculating dates for example getting a date two weeks from now add-date(today(), null, null, 14)```


```You can also add negative numbers to get dates in the past which is useful for things like calculating a person's birthdate date based on how many years/months old they are: add-date(today(), 0-${age_years}, 0-${age_months})```

Reply:
Good call!  Done.